dev-scripts/.get/get-caddy.js

```diff
@@ -51,7 +51,9 @@ exports.getCaddy = function ({ directory, platform, arch }) {
           } else if (response.headers['content-type'] === 'application/x-gzip') {
             req
               .pipe(createGunzip())
-              .pipe(tar.extract(directory)
+              .pipe(tar.extract(directory, {
```

Comment:
Does this change have to do with the shell

Reply:
No it was suppose to go in two commits. (my bad)